### PR TITLE
Backport 10.1.3 - Modify docker compose

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: bamboo/docker-compose.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ services:
     environment:
       SERVICES: "cloudformation,cloudwatch,cloudwatchlogs,dynamodb,iam,kinesis,kms,lambda,s3,secretsmanager,sns,sqs,stepfunctions,ssm,logs"
   build_env:
-    image: maven.earthdata.nasa.gov/cumulus:latest
+    image: ${bamboo_CUMULUS_BASE_IMAGE:-maven.earthdata.nasa.gov/cumulus:latest}
```
Comment (Jkovarik) [inline bamboo/docker-compose.yml:49]: Is the implication that we'd have a base image version for incompat image changes?

Comment (Jkovarik) [reply bamboo/docker-compose.yml:49]: Ah, I see there's a slack thread.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,177 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
+- **CUMULUS-2863**
+  - Fixed `@cumulus/api` `validateAndUpdateSqsRule` method to allow 0 retries and 0 visibilityTimeout
+    in rule's meta.
+
+- **CUMULUS-2959**
+  - Fixed `@cumulus/api` `granules` module to convert numeric productVolume to string
+    when an old granule record is retrieved from DynamoDB
+
+## [v13.0.0] 2022-06-13
```
Comment (Jkovarik) [inline CHANGELOG.md:143]: This doesn't seem right....

**Review by Jkovarik — CHANGES_REQUESTED** [review]
The other changes look good, but the CL seems incorrect.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,177 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+    parameterized and accommodate a new parameter `removeAliasConflict` which
+    allows/disallows the deletion of a conflicting `cumulus-alias` index
+
+### Notable changes
+
+- **CUMULUS-2929**
+  - Updated `move-granule` task to check the optional collection configuration parameter
+    `meta.granuleMetadataFileExtension` to determine the granule metadata file.
+    If none is specified, the granule CMR metadata or ISO metadata file is used.
+>>>>>>> d057f51c6 (update moment to fix security vulnerability (#3025))
```
[the remark this answers is not in the document]
Comment (botanical) [reply CHANGELOG.md]: Sorry about that https://github.com/nasa/cumulus/pull/3033/commits/d71f99e7d538ad6dd07d6a95983af9f02b14637e

File: CHANGELOG.md (hunk 2 above)
Comment (botanical) [reply CHANGELOG.md]: https://github.com/nasa/cumulus/pull/3033/commits/d71f99e7d538ad6dd07d6a95983af9f02b14637e 😞

**Review by Jkovarik — APPROVED** [review]

**Pull request merged**